scripts/pirateofdoom.py

```diff
@@ -43,9 +44,25 @@
     "printing": "scribblin'",
     "load": "haul",
     "connect to": "board",
+    "connecting": "boarding",
     "collects": "hoards",
     "prime tower": "buoy",
     "change log": "captain's log",
     "my": "me",
-    "removable drive": "life boat"
+    "removable drive": "life boat",
+    "print core": "scribbler",
+    "printcore": "scribbler",
```

Comment:
These are not going to be translated. Since they are specific to Ultimaker 3 they are in the UM3 data file and not translated. I'll leave them in for now because it can't hurt.